Engines/Wine/Verbs/quartz/script.js

```diff
@@ -0,0 +1,36 @@
+include(["Engines", "Wine", "Engine", "Object"]);
+include(["Utils", "Functions", "Net", "Resource"]);
+include(["Utils", "Functions", "Filesystem", "Files"]);
+
+Wine.prototype.quartz = function(){
```

Comment:
Maybe add some documentation about this function/verb

Reply:
I don't see well where you want some documentation. It's quite explicit and the other verbs doesn't have any "documentation"

Reply:
You're basically right.
What I've meant is that you add a javadoc like comment to the head of the method.
This comment should contain the content of the verb and its source (ideally with an url).
In this case, the comment could be `Inspired from winetricks quartz` combined with the url to the source of the winetricks quartz implementation.

Reply:
Is what I've just commit what you want ?

Reply:
Almost, I think the link to the winetricks implementation of the script would be better, which I think is: https://github.com/Winetricks/winetricks/blob/63bc6dbe612d017a0cb6bf6e4cde265162d75bca/src/winetricks#L8523

Reply:
Yes, sorry, that wasn't what I wanted to copy. I have used https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks

Reply:
It's corrected now